finance.rb

```diff
@@ -0,0 +1,91 @@
+require 'pry'
+
+class Menu
+  def initialize(payee, amount, date, category)
+    attr_reader :payee
+    @payee = payee
+    @amount = []
+    @date = []
+    @category = []
+    @saving_balance = 0
+    @single_transaction = []
+    @transactions = []
+  end
+
+  def self.display
+      puts "Choose one of the following:"
+      puts "1 - Saving Transactions"
+      puts "2 - Checking Transactions"
+      @@input = gets.chomp
+      if ["1","2"].include? @@input
+        self.select @@input
+      else
+        puts "Invalid option."
+      end
+    end
+
+  def self.select number
+    puts "You selected #{number}"
+    if @@input == "1"
+      Menu.secondisplay
+    end
+  end
+
+  def self.secondisplay
+      puts "Choose one of the following:"
+      puts "1 - Payee"
+      puts "2 - Amount"
+      puts "3 - Date"
+      puts "4 - Category"
+      puts "5 - Transactions"
+      @@input = gets.chomp
+      if ["1","2", "3", "4", "5"].include? @@input
+        self.select2 @@input
```

Comment:
You might use your Transactions class if you add a class car array containing all its instances